### PR TITLE
[PINT-1164] Use default getUrl method

### PR DESCRIPTION
**Pull request by vsavadi**

### CODE REVIEW

File: view/frontend/web/js/view/minicart/fast-button.js [hunk 1]
```diff
@@ -51,9 +51,13 @@ define([
 
         ajaxCall: function(callback){
             var self = this;
+            let url = '/fast/config/fast';
+            if (self.hasOwnProperty('configUrl') && self.configUrl){
```
Comment (corywebb) [inline view/frontend/web/js/view/minicart/fast-button.js:55]: Where is the `configUrl` property getting set?

**Review by corywebb — COMMENTED** [review]
@vsavadi this looks ok, but I'm not sure how/where the `configUrl` property is getting set.

File: view/frontend/web/js/view/minicart/fast-button.js (hunk 1 above)
Comment (vsavadi) [reply view/frontend/web/js/view/minicart/fast-button.js:55]: on the [layout/default.xml](https://github.com/fast-af/magento2/pull/32/files#diff-eede3316d0d7cf919759dbf059e6a1562861b1a64e18c40e07d1d6114941bdaa) file

Comment (vsavadi) [reply view/frontend/web/js/view/minicart/fast-button.js:55]: through xml configuration we can pass variables to js component.

**Review by corywebb — APPROVED** [review]
👍

**Pull request merged**